UnityProject/Assets/Scripts/Systems/Inventory/Pickupable.cs

```diff
@@ -146,29 +147,41 @@ public void ServerRollbackClient(HandApply interaction)
 	}
 
 	public virtual void ServerPerformInteraction(HandApply interaction)
+	{
+		StartCoroutine(serverPreformInteractionLogic(interaction));
+	}
+
+	private IEnumerator serverPreformInteractionLogic(HandApply interaction)
```

Comment:
PascalCase

Reply:
Not my function, not going to change it.

Reply:
it actually is your coroutine. You named it.